src/utils/Xbqcrypto.ts

```diff
@@ -2,24 +2,59 @@ import * as varint from "varint";
 import { blake3 } from "@noble/hashes/blake3";
 const base58check = require("base58check");
 
+/**
+ * Hashes a given data by using the blake3 algorithm
+ *
+ * @param {Uint8Array | string} data the data to apply the blake3 hashing on
+ *
+ * @returns {Uint8Array} the hashed data as a byte array
```

Comment:
You must not write types in typedoc because it take the types declared in the function signature

```suggestion
 * @param data - the data to apply the blake3 hashing on
 *
 * @returns the hashed data as a byte array
```